packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -308,6 +311,8 @@ class ServiceConnectionManager {
     }
 
     _connectionAvailableController.add(service);
+
+    resolvedUriManager.vmServiceOpened();
```

Comment:
call this where all the other managers' vmServiceOpened methods are called 183-190. Same comment for the vmServiceClosed call below